### PR TITLE
DOCSP-44528-cutover-failure-warnings

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Adding cutover warnings 

## STAGING

https://deploy-preview-485--docs-cluster-to-cluster-sync.netlify.app/reference/cutover-process/
https://deploy-preview-485--docs-cluster-to-cluster-sync.netlify.app/reference/mongosync/mongosync-behavior/#considerations-for-continuous-sync

## JIRA

https://jira.mongodb.org/browse/DOCSP-44528

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — COMMENTED** [review]
Thanks for this issue, @ltran-mdb2! This is a great start. I left a question and noted a typo.

Thanks!
Joe

File: source/reference/cutover-process.txt [hunk 1]
```diff
@@ -16,16 +16,33 @@ You can finalize a migration and transfer your application
...
+``mongosync`` should remain active until it reaches the
+:ref:`COMMITTED <c2c-state-committed>` state. This allows ``mongosync`` to sync
+any additional writes that occur during the migration.
+
+.. warning::
+
+   If you do not properly cutover from your source to your
+   destination, you may face the following issues:
+
+   - Garbage data on the destination cluster
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/reference/cutover-process.txt]: I changed to "unstructured and unneeded data" but also sent a slack message to Maria (ticket creator) for more clarity about what she meant by "garbage data"

**Review by jmd-mongo — APPROVED** [review]
Thanks for these updates, @ltran-mdb2! This one LGTM at this point.

Thanks,
Joe

File: source/reference/cutover-process.txt [hunk 2]
```diff
@@ -16,16 +16,33 @@ You can finalize a migration and transfer your application
...
+``mongosync`` should remain active until it reaches the
+:ref:`COMMITTED <c2c-state-committed>` state. This allows ``mongosync`` to sync
+any additional writes that occur during the migration.
+
+.. warning::
+
+   If you do not properly cutover from your source to your
+   destination, you may face the following issues:
+
+   - Unstructured or unneeded data on the destination cluster
```
Comment (mvankeulen94) [inline source/reference/cutover-process.txt:34]: I think we can rephrase to "inaccurate data"

File: source/reference/cutover-process.txt [hunk 3]
```diff
@@ -16,16 +16,33 @@ You can finalize a migration and transfer your application
...
+   - Commit failures with no option to rollback the sync
```
Comment (mvankeulen94) [inline source/reference/cutover-process.txt:35]: I think we can delete the "commit failures with no option [...]" bullet in favor of the unrecoverable and undiagnosable errors bullet below

File: source/reference/cutover-process.txt [hunk 4]
```diff
@@ -16,16 +16,33 @@ You can finalize a migration and transfer your application
...
+   - Unrecoverable and undiagnosable ``mongosync`` errors
+   - Reduced application availability, especially if there is
+     application-side enforcement of document schema or other
+     characteristics
+   - Missing transactions on the destination cluster
```
Comment (mvankeulen94) [inline source/reference/cutover-process.txt:40]: missing -> incomplete

File: source/reference/cutover-process.txt [hunk 5]
```diff
@@ -115,11 +132,16 @@ Steps
...
+      After you submit a ``commit`` request, call the ``progress`` endpoint
+      to ensure that the ``mongosync`` state is ``COMMITTING`` or
+      ``COMMITTED``.
+
+      .. warning::
 
-         After you submit a ``commit`` request, call the ``progress`` endpoint
-         to ensure that the ``mongosync`` state is ``COMMITTING`` or
-         ``COMMITTED``.
+         If the source cluster shuts down before mongosync can commit, such
```
Comment (mvankeulen94) [inline source/reference/cutover-process.txt:141]: can we make this a repeatable snippet so we don't have the same text copied over in 2 places?

File: source/reference/cutover-process.txt [hunk 6]
```diff
@@ -16,16 +16,33 @@ You can finalize a migration and transfer your application
...
+   - Unstructured or unneeded data on the destination cluster
+   - Commit failures with no option to rollback the sync
+   - Unrecoverable and undiagnosable ``mongosync`` errors
+   - Reduced application availability, especially if there is
+     application-side enforcement of document schema or other
+     characteristics
+   - Missing transactions on the destination cluster
+   - Other unexpected cluster behavior
+
+   You must use the following steps to safely finalize your cutover process and achieve
```
Comment (mvankeulen94) [inline source/reference/cutover-process.txt:43]: I suggest rephrasing "You must use the following steps" to "The following steps describe how to safely finalize your cutover process [...]"

**Review by mvankeulen94 — CHANGES_REQUESTED** [review]
thank you! a few changes from my end

File: source/reference/cutover-process.txt (hunk 6 above)
Comment (ltran-mdb2) [reply source/reference/cutover-process.txt]: Done

**Review by edobranov — APPROVED** [review]
LGTM!

**Comment by ltran-mdb2** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v1.9|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/501)|
|✅|v1.8|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/502)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**